content/blog/eu_digital_sovereignty.md

```diff
@@ -0,0 +1,83 @@
+# EU Digital Sovereignty at Risk
+
+## Introduction
+
+Data sovereignty refers to the principle that data is subject to the laws of the country in which it resides.
+
+For individuals, this subject might seem trivial—after all, everyone shares data online at their own discretion. However, the stakes increase drastically when we consider data collected by governments or public institutions. This includes medical records, financial information, and government communications.
+
+Data sovereignty directly affects national security, privacy, and democratic independence. If a foreign government can access digital data from another state, privacy becomes little more than a slogan.
+
+In recent years, the issue has become more urgent. The CLOUD Act and the exponential growth of American tech companies, combined with the scale of European data moving into foreign infrastructures, have reignited the debate: Can Europe ever be fully independent while relying on foreign infrastructure?
+
+## What Are US Companies Putting in Place to Reassure Europe of Their Sovereignty?
+
+The biggest US companies providing cloud services in Europe have come forward with their own "sovereign" solutions to try and reassure Europeans that their data is safe and remains under their control.
+
+### Microsoft
+
+Microsoft has launched sovereign solutions which will ensure that all European data stays in Europe and is controlled by Microsoft employees in Europe. It will also include an encryption controller operated by the customer, meaning the cloud provider will not have the decryption key.
+
+### AWS
+
+AWS is about to launch its version of a dedicated cloud for Europe, with infrastructure and personnel from Europe having access to and control of the data stored in Europe.
+
+### Oracle
+
+Oracle is also launching an EU Sovereign Cloud, claiming it will be a completely separate entity from Oracle US. All data will be stored and controlled in Europe.
+
+All these initiatives show that US companies are attempting to demonstrate a genuine move toward European clouds that are independent of any external control.
+
+## The Legal Reality: Jurisdiction Beats Geography
+
+All these initiatives share one fundamental flaw: under the CLOUD Act, geography does not matter—jurisdiction does.
+
+No matter where the data resides, as long as it is stored on hardware owned by an American company or one of its subsidiaries, that data falls under U.S. jurisdiction. If the U.S. government requests access, the company is legally bound to comply, or face severe consequences such as heavy fines, loss of federal contracts, or even exclusion from doing business in the United States.
```

Comment:
Is this really a hard statement in regards to the US owned hardware?
What about GCP datacenters operated by Thales in France? Maybe they own the GCP hardware? (bought it/leased it?)